pkg/extract/rocky/errata/errata.go

```diff
@@ -154,72 +155,95 @@ func Extract(args string, opts ...Option) error {
 
 func extract(fetched errata.Advisory, raws []string) (dataTypes.Data, error) {
 	ds, err := func() ([]detectionTypes.Detection, error) {
-		ds := make([]detectionTypes.Detection, 0, len(fetched.Rpms))
-		for product, rpms := range fetched.Rpms {
-			cs, err := func() ([]criterionTypes.Criterion, error) {
-				m := make(map[string]map[string][]string)
-				for _, rpm := range rpms.NVRAS {
-					nvr := strings.Split(strings.TrimSuffix(rpm, ".rpm"), "-")
-					if len(nvr) < 3 {
-						return nil, errors.Errorf("unexpected nvras format. expected: %q, actual: %q", "<name>-<version>-<release>.<arch>", rpm)
-					}
-					ra := strings.Split(nvr[len(nvr)-1], ".")
-					if len(ra) < 2 {
-						return nil, errors.Errorf("unexpected nvras format. expected: %q, actual: %q", "<name>-<version>-<release>.<arch>", rpm)
-					}
+		// vm := make(map[string]int)
+		// for _, p := range fetched.AffectedProducts {
+		// 	vm[p.Name] = p.MajorVersion
+		// }
```

Comment:
Not needed?

Reply:
I want to leave it here because it is relevant here.
https://github.com/MaineK00n/vuls-data-update/pull/640/files#diff-3cdc44436b6aca73cf3e55d2755e64570f92491081c04b0dad5eb4e84e4c394cR198-R203